Userland/Libraries/LibWeb/Loader/LoadRequest.h

```diff
@@ -35,6 +35,18 @@ class LoadRequest {
     ByteBuffer const& body() const { return m_body; }
     void set_body(ByteBuffer const& body) { m_body = body; }
 
+    ErrorOr<void> try_set_body(ByteBuffer const& body)
+    {
+        m_body = TRY(ByteBuffer::copy(body.bytes()));
+        return {};
+    }
+
+    ErrorOr<void> try_set_body(ReadonlyBytes const& bytes)
+    {
+        m_body = TRY(ByteBuffer::copy(bytes));
+        return {};
+    }
```

Comment:
This doesn't seem ideal - `set_body()` takes a reference and always copies, so it should just take by value and move. Surely there aren't too many instances to update? :sweat_smile:

Reply:
I am on thin ice when it comes to move. Did some read up on and I think I got how it works now 😅